generator/monitor/monitor.go

```diff
@@ -129,8 +129,20 @@ func pprToStatus(ppr *podseidonv1a1.PodProtector) observer.MonitorWorkloads {
 		))
 	}
 
+	isNonZero := 0
+	if ppr.Spec.MinAvailable > 0 {
```

Comment:
tiny tips, what about isNonZero -> NumNonZeroWorkloads and isFullyAvailable -> NumAvailableWorkloads

Reply:
Do you mean renaming the variable or constructing the struct value and mutating it later?